src/pages/TaskRun/Details-Tile.vue

```diff
@@ -227,7 +243,23 @@ export default {
               <v-col cols="6" class="text-right font-weight-bold">
                 <v-tooltip top>
                   <template #activator="{ on }">
-                    <span v-on="on">
+                    <span
+                      class="cursor-pointer show-icon-hover-focus-only pa-2px"
+                      role="button"
+                      @click="
+                        copyTextToClipboard(
+                          taskRun.serialized_state._result.location
+                        )
+                      "
+                      v-on="on"
+                    >
+                      <v-icon x-small class="mb-2px mr-2" tabindex="0">
```

Comment:
Should we add a v-if here to check if there's actually a result location to copy?

Reply:
Yeah that's sensible - the copy method won't throw an error since it checks for text to copy but I like hiding the icon in the case as well.